src/ElectricPokemon.java

```diff
@@ -0,0 +1,90 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class ElectricPokemon extends Pokemon {
+    private String specialAttack;
+    private String strongAgainst;
```

Comment:
Deze twee variabelen worden in de aplpicatie niet gebruikt en hebben dus geen toegevoegde waarde